### PR TITLE
[promtail] metrics svc labels template bugfix

### PR DESCRIPTION
**Pull request by mjnagel**
Fix for default label issue - bug introduced by https://github.com/grafana/helm-charts/pull/3083

See https://github.com/grafana/helm-charts/pull/3083#issuecomment-2157791678

Closes https://github.com/grafana/helm-charts/issues/3162

### CODE REVIEW

**Comment by mjnagel** [comment]
@zalegrala @zanhsieh this fixes the issue from my previous PR, hopefully a quick one to review/approve 😃

File: charts/promtail/templates/service-metrics.yaml [hunk 1]
```diff
@@ -6,9 +6,13 @@ metadata:
   namespace: {{ include "promtail.namespaceName" . }}
   labels:
     {{- include "promtail.labels" . | nindent 4 }}
-    {{- toYaml .Values.service.labels | nindent 4 }}
+    {{- with .Values.service.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
   annotations:
```
Comment (jkroepke) [inline charts/promtail/templates/service-metrics.yaml:12]: Include annotations into the with here.

Comment (mjnagel) [reply charts/promtail/templates/service-metrics.yaml:13]: Good catch - helm template actually works either way but it does produce a weird yaml, better to just wrap that as well.

**Comment by jimmy-ungerman** [comment]
@zalegrala @zanhsieh this is blocking us as the current chart is in a broken state

**Review by zalegrala — APPROVED** [review]

**Comment by QuentinBisson** [comment]
@zalegrala do we need to do anything to have this released?

**Comment by zalegrala** [comment]
I'm open to extending the list of maintainers if community folks are interested.  Ideally there is a commit history in the repo to show why a person would be a good maintainer.  :)

**Comment by zalegrala** [comment]
Did one of the other PRs handle this?  I see no changes on the PR currently.

**Comment by mjnagel** [comment]
@zalegrala Looks like https://github.com/grafana/helm-charts/pull/3164 got merged in and covered the same changes as this PR.

**Comment by Sheikh-Abubaker** [comment]
> I'm open to extending the list of maintainers if community folks are interested. Ideally there is a commit history in the repo to show why a person would be a good maintainer. :)

I am interested!!

**Pull request closed without merging**